.travis.yml

```diff
@@ -4,4 +4,8 @@ install:
   - gem install puppet-lint
 
 script:
-  - puppet-lint --fail-on-warnings modules
+  - puppet-lint --fail-on-warnings --no-2sp_soft_tabs-check --no-hard_tabs-check --no-names_containing_dash-check --no-autoloader_layout-check --no-nested_classes_or_defines-check modules
+
+notifications:
+  slack:
+    secure: JbcRVXdj9FyVH6kbM0tSYi41+5AfChF4AcN76ibUAMQspqp5xna+cdqXxQiLzG9uzgX4dT/vy1QpTI6pJYtru6N3NhX5lMkfDT2gQr9/h5oHjqv/WStPuCXjsUQIS55MnOcNwNjIwyq6PfiFuX5Zn99YdogCa7mdkgQ5TggujBAexZkXUhnXLdZ+/GsLnUeUkdjeIBHDoYrh+/WYeJ94Fmf9vsgEmP6LULnGwqt4n+tLFkqWS9/q6jirbctl+SlN/UdVuuecozvDAkpm5qB52oEjcZTm7IIjrRGq8Hc8xWNYAxOjKjk56trYRZGYgeo87lYcQ+YHtqb82IZNKD3EuHLtbz1nw/jmEsWEv4nBzoKos/9RXw+9Hzm7NFrooqJwX5ThbdjJNQEsnqloXyehKiOpIznf6OnHoOdFLZQfFiOhyUj3RPrQoCKNE9iTLgLb9QE3NljQZW5GBmvAawj1bEF3xymxBS98JZc8d65kLG3q9iwn06MP2RnD2SOlGLl9ECmNLWB4Hcb3AfGUAMKqaZX/rWmT71NvBvVjBTWZSsYQ4JyIMzpFkxi746ODZzxyk6nR8VbtQtHbC3qZAtS5jQANi56p/eiKyXe3OyDtzmaLyxGbe0hbMb2RpcTXSkn5u/oxd26HFGcuELzKQvC9ygPeo0xl+Y82dQ9+MmbTb+c=
```

Comment:
This should be in a separate PR really (the puppet-lint change is OK to stay though)